gpr2gpa/gpr2gpa.cpp

```diff
@@ -24,16 +24,20 @@
 #include "rapidjson/rapidjson.h"
 #include "rapidjson/document.h"
 
-#include "WindowsUnicodeToolShim.h"
-
 #include "macedec.h"
 
 #include <stdio.h>
 #include <string>
 #include <sstream>
 #include <vector>
 #include <algorithm>
+
+#ifdef _WIN32
+#include "WindowsUnicodeToolShim.h"
 #include "GpWindows.h"
```

Comment:
You can remove the GpWindows.h include here too, it's not used.

Reply:
Done. I also found and removed some other apparently unused Windows-specific includes while refactoring, at least according to `clangd`. I didn't find any platform-specific code hidden by the preprocessor, so `clangd` is probably right about them being unused, but as always, let me know if I've somehow broken the Windows build.